content/product/design/visual_design_checklist.md

```diff
@@ -0,0 +1,34 @@
+# Visual design checklist
+
+This checklist is intended to be suggestions and important considerations when finalizing a design artifact, rather than a prescriptive list of requirements.
+
+This checklist ensures that our designs are consistent, accessible, and optimized for developer handoff.
+
+## Visual design checklist
+
+- Ensure spacing is consistent and matches the 8pt grid system
+- Ensure text, colors and other styles match and use existing styles in Wildcard, if possible.
+  - If adding new styles, ensure that they meet our [accessibility standards](https://handbook.sourcegraph.com/product/design/design-and-interaction-guidelines#accessibility-standards) and add them to Wildcard
+- Review designs for accessibility:
+  - Color blindness check
+  - Contrast check should meet AA standard for small text
+- Create dark mode compositions, if there are not existing dark mode designs in Wildcard
+- Consider preparing redlines or an interaction delivery writeup for the following :
+  - Margin and padding
+  - Animation
+  - Keyboard behavior
+- Due to the low number of mobile and table visitors (< 3%) design comps for these sizes are generally not required
```

Comment:
I think this is an outdated statement. We do not target for mobile and tablet users that much but layouts need to be created in consideration for 1/2 and 1/3 browser windows (which effectively requires us to prepare responsive designs, at least for the main pages). In addition, we've recently decided to put more emphasis on responsiveness at Sourcegraph. My suggestion:

```suggestion
- Provide mockups to support the responsive behaviour. Even though the number of mobile and tablet visitors is relatively low, resizing the browser window to 1/2 and 1/3 width is a common use case. In addition, our objective is to provide a fully responsive platform. Note that there is no need to create a responsive version of every mockup. It's enough to include selected layouts that may be challenging to translate to smaller screen sizes.
```